src/components/elementView/elementTab.tsx

```diff
@@ -318,11 +318,22 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
   };
 
   private createUser = (user: CreateUserObject, headers: any): Promise<any> => {
-    return axios.post(API_ROOT + "api/users/add", user, headers).then((res) => {
-      this.handleDataUpdate(true);
-      this.handleClose();
-      this.addSuccessToast(res.data.success_message);
-    });
+    const username = user.username;
+    return axios
+      .post(API_ROOT + "api/users/add", user, headers)
+      .then((res) => {
+        this.handleDataUpdate(true);
+        this.handleClose();
+        this.addSuccessToast("SUCCESS: User " + username + " created.");
```

Comment:
again, responses from this endpoint are thrown by `django-rest-auth` and not our code, so `success_message` is never sent, so writing the success message here instead